tendermint/Cargo.toml

```diff
@@ -39,8 +39,9 @@ ed25519-dalek = { version = "= 1.0.0-pre.4", features = ["serde"] }
 futures = "0.3"
 k256 = { version = "0.4", optional = true, features = ["ecdsa"] }
 once_cell = "1.3"
-prost-amino = "0.6"
-prost-amino-derive = "0.6"
+prost = "0.6"
+prost-derive = "0.6"
+prost-types = "0.6"
```

Comment:
Is this due to using the well known type for time?

Reply:
I'm not sure what you mean. If you refer to me removing TimeMsg, it's because its implementation was the same or very similar to the built-in prost::Timestamp implementation. We might reintroduce it as a domain-type, though.

Reply:
Ah, I think I understand your question! You are asking why the `prost-types` crate was added!?

It's for `Timestamp`, but I think we might not need it. I'll double-check and remove before we merge this.

Reply:
Yes, I was wondering why the Prost-types dependency was added. Thanks for the clarification.